src/main/resources/META-INF/plugin.xml

```diff
@@ -11,6 +11,11 @@
     <change-notes>
         <![CDATA[
         <p></p>
+        <h3>1.4.0</h3>
```

Comment:
What's this version. Also previous one needs to be fixed

Reply:
ouch